src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/Utils.java

```diff
@@ -33,4 +43,27 @@ public static boolean isNetworkAvailable(final Context context) {
         return connectivityManager.getActiveNetworkInfo() != null && 
             connectivityManager.getActiveNetworkInfo().isConnectedOrConnecting();
     }
+
+    public static void sendFeedEvent(Context context, String category) {
+        Log.d(TAG, "Logging feed event. Category is " + category);
+
+        getTracker(context).send(new HitBuilders.EventBuilder()
+                .setCategory(category)
+                .setAction("click")
+                .build());
+    }
+
+    private static Tracker getTracker(Context context) {
+        if (tracker == null) {
```

Comment:
@myleshorton There is some overlap, but that analytics code is part of the SDK. I don't think it makes sense to intermix tracking events for Everfeed with it (no third-party that integrates Lantern would need any of it). So that's why I keep them separate.